MoreLinq/Permutations.cs

```diff
@@ -24,7 +24,6 @@ namespace MoreLinq
 
     public static partial class MoreEnumerable
     {
-        #region Nested Classes
```

Comment:
> If there's a clean & simple way to deal with this kind of trivia association then this can be reverted.

I don't mind the removal, I was just wondering why it was done. I agree the region was not providing much value anyway.